code/modules/shuttle/ripple.dm

```diff
@@ -11,12 +11,10 @@
 	mouse_opacity = MOUSE_OPACITY_ICON
 	alpha = 0
 
-	duration = 3 * SHUTTLE_RIPPLE_TIME
```

Comment:
can probably remove SHUTTLE_RIPPLE_TIME because i doubt its used anywhere else

Reply:
It is used elsewhere to indicate how long before landing the ripples should be created

Reply:
ah